x-pack/plugins/actions/docs/openapi/components/schemas/secrets_properties_webhook.yaml

```diff
@@ -1,5 +1,23 @@
 title: Connector secrets properties for a Webhook connector
 description: Defines secrets for connectors when type is `.webhook`.
 type: object
-additionalProperties: true
-# TO-DO: Add the properties for this connector.
\ No newline at end of file
+properties:
+  crt:
+    type: string
+    # description: If `authType` is `webhook-authentication-ssl` and `certType` is `ssl-crt-key`, it is an encrypted version of the CRT file?
```

Comment:
All of these are correct, but `base64 encoded` not `encrypted`

Reply:
`.cert` is also a valid extension so maybe describe this as a `CRT or CERT` file

Reply:
Thanks! Fixed in 40dc7ea337e34d50b067d70f0f8ff0fe09048ef5